pkg/config/compaction_lease_controller.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import "time"
+
+const (
+	defaultEnableBackupCompaction = false
+	defaultCompactionWorkers      = 3
+	defaultEventsThreshold        = 1000000
+	defaultActiveDeadlineDuration = 3 * time.Hour
+)
+
+// CompactionLeaseControllerConfig contains configuration for the compaction lease controller.
+type CompactionLeaseControllerConfig struct {
```

Comment:
Instead of creating a config package. I would recommend that we create one per functionality. This will later help in moving parts of the controller into this folder and making the controllers more leaner.